platform/broadcom/sonic-platform-modules-dell/common/dell_i2c_utils.sh

```diff
@@ -60,3 +60,60 @@ i2c_poll_bus_exists() {
     fi
 }
 
+# Perform an i2c mux device create
+# Input is of the form:
+# "echo [mux driver] <i2c-mux-address> >  <i2c-bus/operation>" i2c-channel-first
+# where operation = "new_device"
+# i2c-channel-first is the first of the 8 channels that this mux should create
+i2c_mux_create() {
+    local MAX_MUX_CHANNEL_RETRY=5
+    local MAX_MUX_CHANNELS=8
+    local count=0
+    local i
+    local i2c_bus_op
+    local i2c_bus
+    local i2c_bus_num
+    local i2c_mux
+    local i2c_mux_channel_first
+    local i2c_mux_channel_last
+
+    # Extract the i2c parameters from the command line
+    i2c_bus_op=`echo "$1" | cut -d'>' -f 2`
+    i2c_bus=$(dirname $i2c_bus_op)
+    i2c_bus_num=`echo $(basename $i2c_bus) | cut -d'-' -f 2`
+    i2c_mux=`echo "$1" | cut -d' ' -f 3 | sed 's/^0x//'`
+    i2c_mux_channel_first=$i2c_bus/i2c-$2
+    i2c_mux_channel_last=$i2c_bus/i2c-$(expr $2 + $MAX_MUX_CHANNELS - 1)
+
+    if  i2c_poll_bus_exists  $i2c_bus; then
+        while [[ "$count" -lt "$MAX_MUX_CHANNEL_RETRY" ]]; do
+            eval "$1" > /dev/null 2>&1
+            ret=$?
+
+            # Give more time for the mux channels to get created based on retries
+            i=0
+            while [[ "$i" -lt "$count" ]]; do
+                sleep .5
+                i=$((i+1))
+            done
+
+            # Check if the (first and last) mux channels got created
+            if [[ $ret -eq "0" && -e $i2c_mux_channel_first && -e $i2c_mux_channel_last ]]; then
+                break;
+            else
+                # If the channel did not get created, remove the mux, reset the mux tree and retry
+                i2c_config "echo 0x$i2c_mux > /sys/bus/i2c/devices/i2c-$devnum/i2c-$i2c_bus_num/delete_device"
```

Comment:
could you rename the function `i2c_config` to `i2c_mux_remove`? and make the function symmetric with `i2c_mux_remove`?